resources/frontend_client/index_template.html

```diff
@@ -8,20 +8,15 @@
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
     <link rel="apple-touch-icon" sizes="180x180" href="/app/assets/img/apple-touch-icon.png">
-    <link rel="icon" type="image/png" sizes="32x32" href="/app/assets/img/favicon-32x32.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="/app/assets/img/favicon-16x16.png">
```

Comment:
Without removing these, Firefox was always resolving to the `32x32` one, even if there was a custom link set in the whitelabel settings. Brave and Chrome resolved correctly to the custom one.

I decided to remove them since every browser I tested worked with `favicon.ico`. Also, [the article](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs) that served as the source for these changes never mentioned the need to set these png favicons explicitly. On the contrary.

The bar minimum that works across all browsers is:
```html
<link rel="icon" href="/favicon.ico"><!-- 32×32 -->
<link rel="icon" href="/icon.svg" type="image/svg+xml">
<link rel="apple-touch-icon" href="/apple-touch-icon.png"><!-- 180×180 -->
<link rel="manifest" href="/manifest.webmanifest">
```

Which means we're only missing the SVG icon, but that's more an optimization than a mistake in the config.

Reply:
I like this! As long as there is a cross-browser and browser-compatible way to support favicon without giving a browser-specific `link`, we should do that.